CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+## `Polkadot` projects is a **OPENISH Open Source Project**
+-----------------------------------------
+
+## What?
+
+Individuals making significant and valuable contributions are given commit-access to a project to contribute as they see fit. A project is more like an open wiki than a standard guarded open source project.
+
+## Rules
+
+There are a few basic ground-rules for contributors (including the maintainer(s) of the project):
+
+1. **No `--force` pushes** or modifying the Git history in any way.
```

Comment:
You may want to add that you are talking about the main repo, otherwise, rebases will be hard :)